test/unit/Filters.test.js

```diff
@@ -103,4 +105,48 @@ describe('Filters', () => {
       ).to.equal(true);
     });
   });
+
+  describe('Filter Interactions', () => {
```

Comment:
Still working on the "show error when dates are wrong" test, but I thought I'd push it anyway, this situation is proving a bit tricky to test in an automated way due to the way enzyme's simulate works.